searchcore/src/vespa/searchcore/proton/server/maintenancedocumentsubdb.cpp

```diff
@@ -38,4 +41,10 @@ MaintenanceDocumentSubDB::clear()
     _feed_view.reset();
 }
 
+bool
+MaintenanceDocumentSubDB::lidNeedsCommit(search::DocumentIdT lid) const {
+    return (_pendingLidsForCommit &&
```

Comment:
Consider adding `!= nullptr` for a more standard style with pointer checks.

Reply:
Fixed